spec.html

```diff
@@ -35099,28 +35122,75 @@ <h1>Notation</h1>
+            A <em>Matcher</em> is an Abstract Closure that takes two arguments&mdash;a State and a Continuation&mdash;and returns a MatchResult result. A Matcher attempts to match a middle subpattern (specified by the closure's captured values) of the pattern against the State's _input_, starting at the intermediate state given by its State argument. The Continuation argument should be a closure that matches the rest of the pattern. After matching the subpattern of a pattern to obtain a new State, the Matcher then calls Continuation on that new State to test if the rest of the pattern can match as well. If it can, the Matcher returns the State returned by Continuation; if not, the Matcher may try different choices at its choice points, repeatedly calling Continuation until it either succeeds or all possibilities have been exhausted.
           </li>
         </ul>
+
+        <emu-clause id="sec-regexp-records">
+          <h1>RegExp Records</h1>
+          <p>A <dfn variants="RegExp Records">RegExp Record</dfn> is a Record value used to store information about a RegExp that is needed during compilation and possibly during matching.</p>
+          <p>It has the following fields:</p>
+          <emu-table id="table-regexp-record-fields" caption="RegExp Record Fields">
+            <table>
+              <tr>
+                <th>Field Name</th>
+                <th>Value</th>
+                <th>Meaning</th>
+              </tr>
+              <tr>
+                <td>[[IgnoreCase]]</td>
+                <td>a Boolean</td>
+                <td>indicates whether *"i"* appears in the RegExp's flags</td>
+              </tr>
+              <tr>
+                <td>[[Multiline]]</td>
+                <td>a Boolean</td>
+                <td>indicates whether *"m"* appears in the RegExp's flags</td>
+              </tr>
+              <tr>
+                <td>[[DotAll]]</td>
+                <td>a Boolean</td>
+                <td>indicates whether *"s"* appears in the RegExp's flags</td>
+              </tr>
+              <tr>
+                <td>[[Unicode]]</td>
+                <td>a Boolean</td>
+                <td>indicates whether *"u"* appears in the RegExp's flags</td>
+              </tr>
+              <tr>
+                <td>[[WordCharacters]]</td>
+                <td>a CharSet</td>
+                <td>the characters considered "word characters" for the purposes of `\\b`, `\\B`, `\\w`, and `\\W`</td>
+              </tr>
+              <tr>
+                <td>[[NcapturingParens]]</td>
```

Comment:
NcapturingParens is not a great name. CapturingParensCount?

Reply:
How about `[[CapturingGroupsCount]]`?

Reply:
Works for me.

Reply:
Done.